### PR TITLE
Fix workflows

### PR DESCRIPTION
**Pull request by arturo-seijas**
* Fix metadata key
* Fix headers
* Missing docstrings
* Tools configuration
* Minimal fixtures for CI compatibility
* Disable chaos and load testing

### CODE REVIEW

**Review by nrobinaubertin — APPROVED** [review]

**Review by cbartz — DISMISSED** [review]

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -7,13 +7,3 @@ jobs:
...
-  plugins-test:
-    name: Specific test for the plugin
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - name: Install tox
-        run: |
-          python -m pip install --upgrade pip
-          pip install tox
-      - run: tox -e plugins
```
Comment (cbartz) [inline .github/workflows/test.yaml:19]: Could you please clean up `tox.ini` and remove the `plugins` env? I think it was copied from indico-operator and doesn't apply to this repository.

**Review by cbartz — DISMISSED** [review]
LGTM

**Review by nrobinaubertin — DISMISSED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -1,19 +1,50 @@
...
+
 [tool.pytest.ini_options]
 minversion = "6.0"
 log_cli_level = "INFO"
 
-# Formatting tools configuration
-[tool.black]
-line-length = 99
-target-version = ["py38"]
-
```
Comment (yanksyoon) [inline pyproject.toml:16]: May I know what is the reason for removing this line? From what I understand, this is to have compatibility with VSCode black formatter setup in contributing guide :D

**Review by yanksyoon — APPROVED** [review]
LGTM!

**Review by weiiwang01 — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (cbartz) [reply pyproject.toml:16]: Its still in the file, just moved up.

Comment (arturo-seijas) [reply pyproject.toml:16]: It has been not removed. Now it's on line 10 ;)

**Review by cbartz — APPROVED** [review]

**Pull request merged**